code/ui/manager/src/components/layout/mobile.tsx

```diff
@@ -147,72 +149,100 @@ export interface MobileState {
   active: ActiveTabsType;
 }
 
-class Mobile extends Component<MobileProps, MobileState> {
-  constructor(props: MobileProps) {
-    super(props);
-
-    const { options } = props;
-    this.state = {
-      active: options.isFullscreen ? CANVAS : options.initialActive || SIDEBAR,
-    };
-  }
-
-  render() {
-    const { Sidebar, Preview, Panel, Notifications, pages, viewMode, options } = this.props;
-
-    const { active } = this.state;
-    return (
-      <Root>
-        <Notifications
-          placement={{
-            position: 'fixed',
-            bottom: 60,
-            left: 20,
-            right: 20,
-          }}
-        />
-
-        <Panels active={active} isFullscreen={options.isFullscreen}>
-          <Sidebar />
-          <div>
-            <div hidden={!viewMode}>
-              <Preview showToolbar={options.showToolbar} id="main" viewMode={viewMode} />
-            </div>
-            {pages.map(({ key, route: Route, render: Content }) => (
-              <Route key={key}>
-                <Content />
-              </Route>
-            ))}
+export const Mobile = ({
```

Comment:
The mobile view was such a mess, pages would show up, but then make it impossible to go back to canvas.

This needs more attentions and proper testing.